master/internal/kubernetes/pod.go

```diff
@@ -187,7 +187,7 @@ func (p *pod) createPodSpecAndSubmit(ctx *actor.Context) error {
 
 func (p *pod) receiveResourceCreationFailed(ctx *actor.Context, msg resourceCreationFailed) {
 	ctx.Log().WithError(msg.err).Error("pod actor notified that resource creation failed")
-	p.insertLog(ctx, time.Now(), msg.err.Error())
+	p.insertLog(ctx, time.Now().UTC(), msg.err.Error())
```

Comment:
Yah, I saw. The creation timestamp for pod events.. I'm ok with leaving this one just to save myself a test run since its documented and I imagine k8s has similar internal policies about always using UTC.